config/drupal-8/drupal-8-all-deprecations.php

```diff
@@ -27,7 +27,7 @@
     // the following rule adds unneeded annotations.
     $rectorConfig->services()->remove(AddDoesNotPerformAssertionToNonAssertingTestRector::class);
 
-    $rectorConfig->bootstrapFiles([
-        __DIR__ . '/../drupal-phpunit-bootstrap-file.php'
-    ]);
+//    $rectorConfig->bootstrapFiles([
+//        __DIR__ . '/../drupal-phpunit-bootstrap-file.php'
+//    ]);
```

Comment:
Don't really get this change, but has been a while, so you might also not know :x